### PR TITLE
Support for Select widgets

### PR DESCRIPTION
**Pull request by bittermandel**
This PR adds support for Select widgets used by a ChoiceField.

Design used can be found here: https://tailwindcss.com/components/forms/

Tests are also passing for this field :+1: 

PS: Added a small gitignore for `.vscode` in this PR as well!

### CODE REVIEW

**Comment by bittermandel** [comment]
There's now also some styling for the Select field

**Comment by smithdc1** [comment]
Thanks for this - I'll try and review fully sometime over the weekend. 

For now I've gone ahead and merged the `.gitignore` change in https://github.com/django-crispy-forms/crispy-tailwind/commit/d98a6c2aa6b67b28152129f3c7831405d8475857.

File: crispy_tailwind/templates/tailwind/field.html [hunk 1]
```diff
@@ -13,21 +13,27 @@
...
-            {# otherwise use django rendering with additional classes added #}
-            {% tailwind_field field %}
+        {# otherwise use django rendering with additional classes added #}
+        {% tailwind_field field %}
+    {% endif %}
 
-            {% include 'tailwind/layout/help_text_and_errors.html' %}
+    {% include 'tailwind/layout/help_text_and_errors.html' %}
+
+    </{% if tag %}{{ tag }}{% else %}div{% endif %}>
```
Comment (smithdc1) [inline crispy_tailwind/templates/tailwind/field.html:36]: I'll need to take a closer look at this section here. I _think_ I had the closing `</div>` the wrong side of one the the `{% endif %} tags. My attempt at checkbox/radio then had an additional `</div>` to compensate for this error.

File: crispy_tailwind/templates/tailwind/layout/select.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    {% for value, label in field.field.choices %}
+        {% include "tailwind/layout/select_option.html" with value=value label=label %}
+    {% endfor %}
+</select>
+<div class="pointer-events-none absolute inset-y-0 right-0 flex items-center px-2 text-gray-700">
+    <svg class="fill-current h-4 w-4" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path d="M9.293 12.95l.707.707L15.657 8l-1.414-1.414L10 10.828 5.757 6.586 4.343 8z"/></svg>
+  </div>
+</div>
+{% include 'tailwind/layout/help_text.html' %} 
+</div>
```
Comment (smithdc1) [inline crispy_tailwind/templates/tailwind/layout/select.html:15]: As per previous comment - I think my error has also resulted in this additional closing `div` here.

File: crispy_tailwind/templates/tailwind/field.html (hunk 1 above)
Comment (bittermandel) [reply crispy_tailwind/templates/tailwind/field.html:36]: Yeah, that was it. I moved it out one step and reformatted for readability, so this should be correct!

**Comment by smithdc1** [comment]
This looks great 👍 

Just added an extra `</div>` at the end of the test.

**Comment by bittermandel** [comment]
Good stuff, thanks @smithdc1! Feels good to have my first Open Source PR merged :rocket:

**Pull request merged**